### ISSUE TITLE
Torrents added to rtorrent being set to Paused

### ISSUE THREAD

**Issue by joshoram80** [JohnDoee/autotorrent#31]
I have your script setup and it seems to work perfectly other than the fact that torrents added to rtorrent always get set to Paused after a few seconds of seeding. Config is below. 

```
[general]
db = _autotorrent.db
ignore_files =
store_path = store_paths/seeds/
add_limit_size = 64000000
add_limit_percent = 1
link_type = soft
scan_mode = unsplitable,normal

[disks]
disk1 = /media/dmm/spicywiener/private/rtorrent/data/TV
disk2 = /media/dmm/spicywiener/private/rtorrent/data/Movies
disk3 = ~/pip/bin/store_paths/seeds/

[client]
url = scgi:///media/dmm/spicywiener/private/rtorrent/.socket
client = rtorrent
label = autotorrent
```

**Comment by d2dyno** [comment]
Sounds more like an rtorrent configuration. Do you have the setting enabled, that adds torrents paused?

**Comment by joshoram80** [comment]
Nope. The torrents are Started when they are added and seed for a while before pausing. I can restart seeding and they go again for 10-15 seconds and Pause again.

Only torrents added using Autotorrent do this. I have Radarr and Medusa setup and they work fine.

**Comment by JohnDoee** [comment]
Is autotorrent running as the same user as rtorrent?

The pause stuff is how rtorrent can behave when it does not have write access to the files if it expects to download e.g. a missing NFO.

**Comment by joshoram80** [comment]
Apparently it is. ps aux shows my username as running rtorrent, and autotorrent is run from a venv in my home folder, owner is set to me as well. I'm not sure how else to check but it seems ok. It does actually seed but only for 15 seconds then pauses. The files are 100% identical so it shouldn't be downloaded anything missing.

**Comment by JohnDoee** [comment]
Can you check the links in `store_paths/seeds/` actually points to the correct files? The config option `disk3 = ~/pip/bin/store_paths/seeds/` might confuse it.

**Comment by joshoram80** [comment]
I removed that disk3 line, and yes the links point to the correct location

**Comment by JohnDoee** [comment]
You might need to enable rtorrent to figure out what is really going on. You can find an example here https://plaza.quickbox.io/t/where-is-rtorrent-log-file/1865

rtorrent isn't the easiest thing to debug, sadly.

**Comment by joshoram80** [comment]
[rtorrent.log](https://github.com/JohnDoee/autotorrent/files/3254856/rtorrent.log)
[storage.log](https://github.com/JohnDoee/autotorrent/files/3254857/storage.log)
[tracker.log](https://github.com/JohnDoee/autotorrent/files/3254858/tracker.log)

I've attached logs. storage log seems interesting. I'm not sure why that's happening. If I manually add the torrent it doesn't do that.

rtorrent.log shows
```
1559690908 I 23CEB2DE4B43B103173D212AC549AFFC30E66E13->download_list: Pausing download: flags:0.
1559690908 I 23CEB2DE4B43B103173D212AC549AFFC30E66E13->download: Stopping torrent: flags:0.
```
but doesn't really give a reason why that I can see

**Comment by JohnDoee** [comment]
You're right that it actually works for some time, the logs say that you actually uploaded some data.

Another reason can be a schedule in your .rtorrent.rc file, it would look similar to

`schedule = untied_directory,5,60,stop_untied= ` or something else with untied.

A hint is that the torrent is allowed to run for 60 seconds. This would all be related to a watch directory setup of some kind.

**Comment by joshoram80** [comment]
OK so that's all working now. I have an issue still when autotorrent is run from a script that is run when a download is completed (via rtorrent.rc entry). Autotorrent seems to create the folder and symlinks as it should but the torrent does not get added to rtorrent.

If I run autorrent manually after this it says
`[Exists]    'XXXXXXXXXXXXXX' The folder exist, but is not seeded by torrentclient`

Manually deleting this folder and then running Autorrent again, the torrent is added correctly. By this time it is usually too late and most of the benefit of cross seeding is lost. Everything works perfectly when done manually but automation is given me a headache.

The script that is run by rtorrent is only 2 lines, one to rebuild database and the other to add torrents. I have noticed that I have to use the -c parameter when running autorrent from anywhere but cd'd into the directory where it is located.

**Comment by JohnDoee** [comment]
My guess would be that you have something that removes the torrent after autotorrent adds it. The process is like this:

* Connect to torrent client and check if torrent is already added
* If not added, create folders
* Add torrent

> I have noticed that I have to use the -c parameter when running autorrent from anywhere but cd'd into the directory where it is located.
Yeah, no global user config, mostly because earlier it was useful to have multiple config files.
